telluric/util/general.py

```diff
@@ -2,8 +2,10 @@
 
 
 def convert_meter_to_latlon_deg(lat_deg):
```

Comment:
why did we implement it ourselves? isn't there a 3rd party that calculates this?

Reply:
This was here already, I only improved the documentation - but maybe we can just reproject, let me check

Reply:
No, I don't think there's a way to perform this computation by reprojecting and I'm not aware of any 3rd party library that includes this function - which doesn't mean there isn't one.